cmd/karmada-search/app/options/options.go

```diff
@@ -69,6 +72,8 @@ func (o *Options) AddFlags(flags *pflag.FlagSet) {
 
 	flags.Float32Var(&o.KubeAPIQPS, "kube-api-qps", 40.0, "QPS to use while talking with karmada-apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags.")
 	flags.IntVar(&o.KubeAPIBurst, "kube-api-burst", 60, "Burst to use while talking with karmada-apiserver. Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags.")
+	flags.BoolVar(&o.DisableSearch, "disable-search", o.DisableSearch, "turn off the search feature.")
+	flags.BoolVar(&o.DisableProxy, "disable-proxy", o.DisableProxy, "turn off the proxy feature.")
```

Comment:
```suggestion
	flags.BoolVar(&o.DisableSearch, "disable-search", false, "Disable search feature that would save memory usage significantly.")
	flags.BoolVar(&o.DisableProxy, "disable-proxy", false, "Disable proxy feature that would save memory usage significantly.")
```
Just give my comments here, if we can't combine the two flags together for now.

By the way, what if both flags are `true`? What's the behavior?

Reply:
> By the way, what if both flags are `true`? What's the behavior?

Both search and proxy are working, they have two independent cache. For some large scale cluster, `karmada-search` may OOM.